cli/sync_service/sync_service.go

```diff
@@ -618,21 +561,29 @@ func checkCSSStatus(org string, timeout int) error {
 		c <- fmt.Sprintf("%d", respCode)
 	}()
 
-	for {
-		select {
-		case httpCodeString := <-c:
-			httpCode, err := strconv.Atoi(httpCodeString)
-			cliutils.Verbose(msgPrinter.Sprintf("Received HTTP code: %d", httpCode))
-
-			if err == nil {
-				if httpCode == 200 {
-					return nil
+	checkLimit := 1
+	for checkLimit > 5 {
```

Comment:
Should it be `checkLimit <= 5`

Reply:
Ha! yep, missed that section in my tests.